sweddit_combined/history/countries/CSH - Shun.txt

```diff
@@ -1,7 +1,7 @@
 government = monarchy
-add_government_reform = chinese_warlord
+add_government_reform = confucian_state_reform
```

Comment:
Why is this reform set here but not on all other confucianism warlords?

Reply:
The ones that are set exist at start of game